index.html

```diff
@@ -12,28 +12,9 @@
         <div class="header__logo">
           <h1>Monster Finder</h1>
         </div>
-        <form class="search" id="search-monster-form">
```

Comment:
A form is not required here since all we need is access to the input field which can be done directly